src/components/WalletSelector/multichain/supportedAuthMethods.ts

```diff
@@ -1,17 +1,21 @@
-export type SupportedAuthMethodKey = 'ethereum' | 'gnosisSafe';
+export type SupportedAuthMethodKey = 'ethereum' | 'gnosisSafe' | 'tezos';
```

Comment:
nit: could we consolidate usage of these consts with the one below?

<img width="793" alt="image" src="https://user-images.githubusercontent.com/12162433/186968792-563b9091-367d-4077-aa72-893e4e20380f.png">